src/components/xy_chart/utils/text_utils.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+/**
+ * Simplified version of tspan wrapper that takes an array of Strings.
+ * and return a SVG component composed by tspan tags.
+ * @param {Array of Strings} texts - an array of splitted text, one per line
+ * @return {Object} Returns an Object to use with dangerouslySetInnerHTML
+ * with the rendered markdown HTML
+ */
+function tspanTextWrapper(texts) {
+  return (
+    <tspan>
```

Comment:
should this be a `<text>` element instead?

Reply:
The `<text>` element already wrap the tick text. I've fixed this adding a Fragment instead of a tspan